f3_test.go

```diff
@@ -195,12 +195,13 @@ func TestF3DynamicManifest_WithRebootstrap(t *testing.T) {
 	// check that it rebootstrapped and has a new base epoch.
 	targetBaseEpoch := env.manifest.BootstrapEpoch - env.manifest.EC.Finality
 	env.waitForCondition(func() bool {
+		env.clock.Add(env.manifest.EC.Period)
```

Comment:
We now get the manifest immediately so `waitForManifest` will return long before the bootstrap epoch. So we need to wait longer here.